assets/monero.conf.template

```diff
@@ -1,41 +1,41 @@
 #
 # monero.conf configuration file.  Things beginning with # are comments.
 #
-# See https://monerodocs.org/interacting/monerod-reference/
+# See https://getmonero.dev/interacting/monerod
 
 # Do not require tty as in foreground mode: "Helpful when running in a container."
 # This keeps monero from crashing because no tty is assigned.
 # We are commenting it here because it *must* be invoked from the command line via `monerod --non-interactive`
 #non-interactive=1
 
 # Data directory (blockchain db and indices)
-data-dir=/data/.bitmonero  # Remember to create the monero user first
+data-dir=BITMONERO_DIR          # Remember to create the monero user first
 
 # Log file
 log-level=0,blockchain:INFO
-log-file=/data/.bitmonero/monero.log
-max-log-file-size=10000000            # Currently 10MB. 0 = Prevent monerod from managing the log files, if we want logrotate to take care of that
+log-file=MONERO_LOG
+max-log-file-size=10000000     # Our default: 10MB. 0 = Prevent monerod from managing the log files, if we want logrotate to take care of that
 max-log-files=2
 
 # P2P full node
-p2p-bind-ip=0.0.0.0            # Bind to all interfaces (the default)
-p2p-bind-port=18080            # Bind to default port
+p2p-bind-ip=MONEROD_BIND_IP            # Bind to all interfaces (the default)
+p2p-bind-port=MONERO_P2P_PORT            # Bind to default port
 
-out-peers=ADV_P2P_MAXNUMOUTPEERS    # This will enable much faster sync and tx awareness; the default 8 is suboptimal nowadays [defacto default: 64]
-in-peers=ADV_P2P_MAXNUMINPEERS      # The default is unlimited; we prefer to put a cap on this ["suggested": 128]
+out-peers=ADV_P2P_MAXNUMOUTPEERS         # Larger values will enable much faster sync and tx awareness
+in-peers=ADV_P2P_MAXNUMINPEERS          # The default is unlimited; we prefer to put a cap on this ["suggested": 128]
 
-limit-rate-up=RATELIMIT_KBPSUP       # Default 2048 kB/s; contribute more to p2p network
-limit-rate-down=RATELIMIT_KBPSDOWN   # Default 8192 kB/s; allow for faster initial sync
+limit-rate-up=RATELIMIT_KBPSUP        # Default 2048 kB/s; contribute more to p2p network
+limit-rate-down=RATELIMIT_KBPSDOWN      # Default 8192 kB/s; allow for faster initial sync
 
 # RPC open node
-rpc-bind-ip=0.0.0.0            # Unrestricted RPC IP and port. 0.0.0.0=Bind to all interfaces [So we can connect a wallet on the LAN]
-rpc-bind-port=18081            # Bind on default port
+rpc-bind-ip=MONEROD_BIND_IP            # Unrestricted RPC IP and port. 0.0.0.0=Bind to all interfaces [So we can connect a wallet on the LAN]
+rpc-bind-port=MONERO_RPC_PORT            # Bind on default port
 confirm-external-bind=1        # Confirm we want the unrestricted RPC (set by rpc-bind-ip) listening on all network interfaces
```

Comment:
```suggestion
#confirm-external-bind=1        # Confirm we want the unrestricted RPC (set by rpc-bind-ip) listening on all network interfaces
```
we don't

Reply:
Do you realize this is running in a docker container? If it doesn't listen on the docker rfc1918 address, it's not reachable from the network.  This doesn't mean monerod's RPC is exposed to the LAN or the internet.

Reply:
Are is there a reason / function that relies on the unrestricted rpc?

Reply:
You cannot bind monero's rpc port to 0.0.0.0 unless you also include `confirm-external-bind=1` -- since we need to bind the RPC port to 0.0.0.0 in order to make it accessible from other containers and LAN/onion wallets, we must include confirm-external-bind.

We expose the unrestricted RPC as well as the restricted RPC to the other containers (and Tor) so that anybody with any use case can use the RPC they desire.  Out of curiosity, why are you so against exposing the unrestricted RPC?  Any user who wishes can password-restrict the RPC in the web-UI config for monero, and it's up to the user if they want to share the tor address with a 3rd party.  No one can discover it unless you give it to them.

Reply:
> Any user who wishes can password-restrict the RPC in

password restricted rpc puts a password on both restricted and unrestricted rpc

Reply:
> password restricted rpc puts a password on both restricted and unrestricted rpc
Yeah, I agree that's lame!

> unrestricted rpc should not be exposed to lan or onion.
OK sounds like a fair assessment, thanks for spelling it out.  I'll put it on localhost or shut it off completely.